src/api/subscriptions.js

```diff
@@ -428,7 +440,7 @@ export let SmartButtonOptions;
  *
  *  @typedef {{
  *    skuId: string,
- *    oldSkuId: (string|undefined),
+ *    oldSku: (string|undefined),
```

Comment:
are you sure nobody is using the key oldSkuId already? Looks like an API change to me.

Reply:
Yeah, doing a quick codesearch, it doesn't show up anywhere that wouldn't have been hidden behind an experimental flag.